### PR TITLE
Suggest ` (backtick) in confusing characters

### PR DESCRIPTION
**Pull request by ishitatsuyuki**
Intended usecase (with current/wrong output):

```
error: unknown start of token: `
  --> src/bn.rs:26:2
   |
17 | [`BIGNUM`]: https://wiki.openssl.org/index.php/Manual:Bn_internal(3)
   |  ^
help: Unicode character '`' (Grave Accent) looks like ''' (Single Quote), but it is not
   |
17 | ['BIGNUM`]: https://wiki.openssl.org/index.php/Manual:Bn_internal(3)
   |  ^
```

### CODE REVIEW

**Review by zackmdavis — CHANGES_REQUESTED** [review]
r=me conditional on removing the Grave Accent line 187

File: src/libsyntax/parse/lexer/unicode_chars.rs [hunk 1]
```diff
@@ -190,6 +184,13 @@ const UNICODE_ARRAY: &[(char, &str, char)] = &[
     ('❞', "Heavy Double Comma Quotation Mark Ornament", '"'),
     ('❝', "Heavy Double Turned Comma Quotation Mark Ornament", '"'),
 
+    ('`', "Grave Accent", '`'),
```
[the remark this answers is not in the document]
Comment (ishitatsuyuki) [reply src/libsyntax/parse/lexer/unicode_chars.rs]: Wait, that brings another question: why is this warning issued at all. It seems the intended usecase in the description is a rustdoc bug or something, as it's replacing seemingly valid Markdown with a more broken one.

**Comment by ishitatsuyuki** [comment]
Closing as obsolete. It was a missing \`\`\` for doctest.

**Pull request closed without merging**